src/commands/build/wranglerjs/mod.rs

```diff
@@ -198,6 +196,17 @@ fn setup_build(target: &Target) -> Result<(Command, PathBuf, Bundle), failure::E
     // if webpack_config is not configured in the manifest
     // we infer the entry based on {package.json} and pass it to {wranglerjs}
     if let Some(webpack_config_path) = custom_webpack_config_path {
+        let mut webpack_file = File::open(&webpack_config_path)?;
+        let mut webpack_buffer = String::new();
+        webpack_file.read_to_string(&mut webpack_buffer)?;
+        if webpack_buffer.contains("wasm-pack-plugin") {
```

Comment:
technically this works but it's... janky

Reply:
I'm wondering if we just.. maybe.. shouldn't install wasm_pack for webpack projects? Is it possible to make a change to the plugin source to instead install wasm_pack as a built binary instead of from source so we don't have to do it in Wrangler? It seems strange to me that we cater to a specific plugin (especially when we don't have a documented template/tutorial for using this specific plugin).